app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -2393,7 +2393,22 @@ public void onDateSet(DatePicker view, int year, int month, int dayOfMonth) {
                 datePickerDialog.show();
                 break;
             case R.id.logout:
-                logout();
+                AlertDialog.Builder alertDialog = new AlertDialog.Builder(this);
+                alertDialog.setTitle(getString(R.string.logout));
+                alertDialog.setMessage(getString(R.string.logout_title));
+                alertDialog.setPositiveButton(getString(R.string.dialog_positive_btn_text), new DialogInterface.OnClickListener() {
```

Comment:
You can use `getString(android.R.string.yes)`  and so on.

http://stackoverflow.com/q/2631278/5612089